Add instruction how to install scala-cli with brew/apt/yum (#100)
VirtusLab/scala-cli: website/docs/installation.md

> move scala-cli.exe "%USERPROFILE%/scala-cli"
```

### Alpine 

Download the launcher from GitHub release assets with

```text
$ wget -q -O scala-cli.gz  https://github.com/VirtuslabRnD/scala-cli/releases/download/nightly/scala-cli-x86_64-pc-linux-static.gz && gunzip scala-cli.gz
$ chmod +x scala-cli
$ mv scala-cli /usr/bin/
```

Check that it runs fine by running its `about` command:
```text
$ scala-cli about
```

## OS-specific packages

### Debian (x86-64)

> #### Apt based installation

Reviewer: ```suggestion
#### Apt-based installation
```

```text
$ curl -s --compressed "https://virtuslabrnd.github.io/scala-cli-packages/KEY.gpg" | sudo apt-key add -
$ sudo curl -s --compressed -o /etc/apt/sources.list.d/scala_cli_packages.list "https://virtuslabrnd.github.io/scala-cli-packages/debian/scala_cli_packages.list"
$ sudo apt update
$ sudo apt install scala-cli
```

> #### Deb package based installation

Reviewer: ```suggestion
#### dpkg-based installation
```

> Alternatively, just download and install the latest build of Debian package with
> 
> [Download Debian package](https://github.com/VirtuslabRnD/scala-cli/releases/download/nightly/scala-cli-x86_64-pc-linux.deb)

Reviewer: ```suggestion
The Debian package can be downloaded at [this address](https://github.com/VirtuslabRnD/scala-cli/releases/download/nightly/scala-cli-x86_64-pc-linux.deb).

Alternatively, you can download it and install it manually with:
```

```text
$ curl -fLo scala-cli.deb https://github.com/VirtuslabRnD/scala-cli/releases/download/nightly/scala-cli-x86_64-pc-linux.deb
$ dpkg -i scala-cli.deb
```

### RPM (x86-64)

> #### Yum based installation

Reviewer: ```suggestion
#### Yum-based installation
```

```text
$ cat > /etc/yum.repos.d/virtuslab.repo << EOF
[virtuslab-repo]
name=VirtusLab Repo
baseurl=https://virtuslabrnd.github.io/scala-cli-packages/fedora/Packages
enabled=1
gpgcheck=1
gpgkey=https://virtuslabrnd.github.io/scala-cli-packages/KEY.gpg
EOF
$ yum repo-pkgs virtuslab-repo list
$ yum install scala-cli    
```

> #### Rpm package based installation

Reviewer: ```suggestion
#### Rpm-based installation
```

> 
> Alternatively, just download and install the latest build of RedHat package with
> 
> [Download RPM package](https://github.com/VirtuslabRnD/scala-cli/releases/download/nightly/scala-cli-x86_64-pc-linux.rpm)

Reviewer: ```suggestion

The RPM package can be downloaded at [this address](https://github.com/VirtuslabRnD/scala-cli/releases/download/nightly/scala-cli-x86_64-pc-linux.rpm).

Alternatively, you can download it and install it manually with:
```

### macOS (brew)

> The scala-cli can be installed via [homebrew](https://brew.sh) with

Reviewer: ```suggestion
Scala CLI can be installed via [homebrew](https://brew.sh) with
```
